code/modules/surgery/bodyparts/parts.dm

```diff
@@ -69,6 +70,12 @@
 		cavity_item = null
 	return ..()
 
+/obj/item/bodypart/chest/proc/get_butt_sprite()
+	return butt_sprite || (owner.gender == FEMALE ? BUTT_SPRITE_HUMAN_FEMALE : BUTT_SPRITE_HUMAN_MALE)
+
+/obj/item/bodypart/chest/felinid
+	butt_sprite = BUTT_SPRITE_CAT
```

Comment:
If we're being real, felinids shouldn't need their own chest bodypart, `get_butt_sprite` should just check for if the mob has a cat tail organ